examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLReadParquetExample.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.examples.sql
+
+import org.apache.spark.sql.SparkSession
+
+// Example/test code: reads a Parquet file from the provided Spark examples
+// Optional argument: path to dir with examples (typically SPARK_HOME)
+// This is also used by the Kubernetes Tests Suite
+object SparkSQLReadParquetExample {
```

Comment:
Can you use the existing `SparkSQLExample` for this? Seems overkill to add a new example just so you can run it in the k8s tests.

I was hoping there was something simple like setting `spark.io.compression.codec=snappy`, but I'm not sure SparkPi would actually hit that. (Maybe `SparkRemoteFileTest`, which is run already, does, though?)

If the above doesn't work, or `SparkSQLExample` is too slow or doesn't work for this, then I think in this case it's better to skip the test until we can separate examples from k8s integration tests.

Reply:
Good point, thanks. Setting `spark.io.compression.codec=snappy` seems to work to test this.